dags/sustainment/check_datastore_resource_description.py

```diff
@@ -0,0 +1,59 @@
+
+from datetime import datetime, timedelta
+import requests
+import logging
+
+from airflow import DAG
+from airflow.models import Variable
+from airflow.operators.python import PythonOperator
+from ckan_operators.package_operator import GetCkanPackageListOperator
+from ckan_operators.datastore_operator import CheckCkanResourceDescriptionOperator
+from utils_operators.slack_operators import GenericSlackOperator, task_failure_slack_alert
+
+# init hardcoded vars for these dags
+ACTIVE_ENV = Variable.get("active_env")
+CKAN_CREDS = Variable.get("ckan_credentials_secret", deserialize_json=True)
+CKAN = CKAN_CREDS[ACTIVE_ENV]["address"]
+CKAN_APIKEY = CKAN_CREDS[ACTIVE_ENV]["apikey"]
+
+with DAG(
+    'check-datastore-description',
+    # These args will get passed on to each operator
+    # You can override them on a per-task basis during operator initialization
+    default_args={
+                "owner": "Yanan",
+                "depends_on_past": False,
+                "email": "yanan.zhang@toronto.ca",
+                "email_on_failure": False,
+                "email_on_retry": False,
+                "retries": 2,
+                "retry_delay": 3,
+                "on_failure_callback": task_failure_slack_alert,
+                "tags": ["dataset", "yaml"]
```

Comment:
This function is good so far, but this "tags" line should be different 

(of course, this is somethign else I havent told you about yet 😅 )

"tags" in Airflow are things we use to organize DAGs. This job of yours would be characterized as a "sustainment" DAG

We can add other tags to this that you think would be helpful, too

Reply:
got it! I'll change it to "sustainment". 👍